doc/api/async_hooks.md

```diff
@@ -712,6 +712,75 @@ never be called.
 * Returns: {number} The same `triggerAsyncId` that is passed to the
 `AsyncResource` constructor.
 
+## Class: AsyncStorage
```

Comment:
can this be renamed to AsyncContext? storage tends to imply some level of persistence, and the term "context" is already used a bunch in describing them as its a fairly apt term.

Reply:
Good one. I was not fan of AsyncClosure but context lgtm